test/plugin/base_test.rb

```diff
@@ -270,23 +270,23 @@ def setup
     }
   }
 
-  # Almost the same as from metadata, but missing namespace_id and pod_id.
+  # Almost the same as from metadata, but namespace_id and pod_id come from
+  # namespace and pod names.
   CONTAINER_FROM_TAG_PARAMS = {
     resource: {
       type: CONTAINER_CONSTANTS[:resource_type],
       labels: {
         'cluster_name' => CONTAINER_CLUSTER_NAME,
+        'namespace_id' => CONTAINER_NAMESPACE_NAME,
         'instance_id' => VM_ID,
+        'pod_id' => CONTAINER_POD_NAME,
         'container_name' => CONTAINER_CONTAINER_NAME,
         'zone' => ZONE
       }
     },
     log_name: CONTAINER_CONTAINER_NAME,
     project_id: PROJECT_ID,
     labels: {
-      "#{CONTAINER_CONSTANTS[:service]}/namespace_name" =>
```

Comment:
You were right -- @summit found some uses of these, so we're leaving them in. Pushed a new commit. PTAL.